src/main/java/com/keniu/services/UserServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.keniu.services;
+
+import com.keniu.dto.CreateUserRequestDto;
+import com.keniu.dto.UserDto;
+import com.keniu.mappers.UserMapper;
+import com.keniu.repositories.UserRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class UserServiceImpl implements UserService {
+
+    private final UserRepository userRepository;
+    private final UserMapper userMapper;
+
+    @Override
+    public UserDto save(CreateUserRequestDto createUserRequestDto) {
+        return userMapper.toDto(userRepository.save(userMapper.toModel(createUserRequestDto)));
```

Comment:
better use variables for improving readability

Reply:
We have already discussed this in another PR. I don't want to repeat myself